packages/desktop-client/src/components/manager/subscribe/OpenIdForm.tsx

```
@@ -236,6 +218,7 @@
             isLoading={loading}
             onPress={onSubmit}
             isDisabled={submitButtonDisabled}
+            style={isNarrowWidth ? { padding: 10} : undefined}
```

ESLint

[error] Insert `·`

(prettier/prettier)